tools/c7n_gcp/c7n_gcp/resources/mlengine.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_gcp.provider import resources
+from c7n_gcp.query import QueryResourceManager, TypeInfo
+
+
+@resources.register('ml-location')
+class MLLocation(QueryResourceManager):
```

Comment:
use case? one doesn't simply walk into mordor, or create locations... ie. nothing to manage or govern.

Reply:
done, could you please recheck?